package.json

```diff
@@ -5,13 +5,15 @@
   "description": "React's PropTypes, packaged up as a standalone module.",
   "main": "index.js",
   "scripts": {
-    "build": "babel -f $npm_package_main -m umd --module-id $npm_package_amdName -s -o $npm_package_main < src/index.js",
+    "build": "babel --module-id $npm_package_amdName -s -d . src",
```

Comment:
Looks good to me: both modules have `define("PropTypes"…` which I suppose is desired behavior.